base/src/test/java/io/fabric8/launcher/base/identity/IdentityFactoryTest.java

```diff
@@ -15,6 +15,7 @@
     public void testTokenIdentity() {
         TokenIdentity identity = IdentityFactory.createFromToken("FOO");
         Assert.assertThat(identity.getToken(), equalTo("FOO"));
+        Assert.assertThat(identity.getType().isPresent(), equalTo(false));
```

Comment:
Can we use `assertj` assertions instead? I believe some of the tests are already based on that. Makes it nicer to read and easier to write.